bundle/Controller/Admin/ImportUrlsController.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace Novactive\Bundle\eZSEOBundle\Controller\Admin;
+
+use EzSystems\EzPlatformAdminUiBundle\Controller\Controller;
+use Novactive\Bundle\eZSEOBundle\Form\Type\ImportUrlsType;
+use Symfony\Component\HttpFoundation\BinaryFileResponse;
+use Symfony\Component\HttpFoundation\File\Stream;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\ResponseHeaderBag;
+use Symfony\Component\Routing\Annotation\Route;
+use Pagerfanta\Adapter\ArrayAdapter;
+use Pagerfanta\Pagerfanta;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+
+class ImportUrlsController extends Controller
+{
+    const URL_LIMIT = 10;
+
+    /**
+     * @param Request $request
+     * @Route("/url-redirect-import", name="novactive_platform_admin_ui.import-redirect-url")
+     * @return Response
+     */
+    public function importAction(Request $request)
+    {
+        if (!$this->get('security.authorization_checker')->isGranted('IS_AUTHENTICATED_REMEMBERED')) {
+            throw new AccessDeniedException('Limited access !!!');
+        }
+
+        $params = $message = [];
+        $importUrlHelper = $this->container->get('novactive_ezseobundle_importurls_helper');
+        $session = $request->getSession();
```

Comment:
this is weird to use the session for that, cannot you just use the Post?

Reply:
I used the session to always display the last recap of the import

Reply:
ok I still don't get why it would not be the post. do you need to keep the recap all the time?

Reply:
yes, we need to keep and display the last recap